lib/src/scaffold.dart

```diff
@@ -572,6 +578,20 @@ class BackdropScaffoldState extends State<BackdropScaffold>
         ),
       ),
     );
+    if (widget.frontLayerBoxShadow == null ||
+        widget.frontLayerBoxShadow!.isEmpty) {
+      return frontPanel;
+    }
+    return _wrapInShadow(frontPanel);
+  }
+
+  Widget _wrapInShadow(Widget frontPanel) {
+    return Stack(children: <Widget>[
+      Container(
+        decoration: BoxDecoration(boxShadow: widget.frontLayerBoxShadow),
+      ),
+      frontPanel,
+    ]);
```

Comment:
Why is `Stack` needed here? Does simply wrapping `frontPanel` in a `Container` with a `decoration` not suffice?

Reply:
oh yeah, you're right, I'll make the change